### ISSUE TITLE
background colour for <element id="PlayerFlat.DisplayPanel">

### ISSUE THREAD

**Issue by karbock** [nopeless/doki-theme-musicbee#2]
TL;DR: 
Because of the way MusicBee v3.6 interprets skins (and this seems to be a correction compared with v3.5), one of the colour references must be replaced in all the skins of the package. The corrected versions work with both MB 3.5 and MB 3.6.
-------
TROUBLESHOOTING
The package in version 0.1.3 works fine under MusicBee v3.5, but in v3.6 the Player Panel shows a grey block where the current track title should be visible. This means that the wrong background colour is taken for that area. The precise element concerned is bg within element id="PlayerFlat.DisplayPanel".

CORRECTION TO APPLY
* current state:
element id="PlayerFlat.DisplayPanel" bg="Playerpanel_Controls_Lowlight"
* corrected line:
element id="PlayerFlat.DisplayPanel" bg="Playerpanel_Background"

TESTS
After applying the patch and testing several skins with MB v3.5 and MB v3.6, the corrected skins work fine in both MusicBee versions.

**Comment by nopeless** [comment]
I'll push a fix soon, thanks for letting me know 👍

**Comment by nopeless** [comment]
Moved to #3
